crypto/evp/evp_lib.c

```diff
@@ -355,7 +355,7 @@ int evp_cipher_cache_constants(EVP_CIPHER *cipher)
                                          &multiblock);
     params[8] = OSSL_PARAM_construct_end();
     ok = evp_do_ciph_getparams(cipher, params);
-    if (ok) {
+    if (ok > 0) {
```

Comment:
Your c-fu is weak today :)
The returned ok value (line 379) needs the check also?

Reply:
Who is Brian :)

Reply:
Exactly.